sidechain/consensus/slots/src/lib.rs

```diff
@@ -268,7 +268,7 @@ impl<B: ParentchainBlock, T: SimpleSlotWorker<B>> PerShardSlotWorkerScheduler<B>
 
 			// important to check against millis here. We had the corner-case in production
 			// setup where `shard_remaining_duration` contained only nanos.
-			if shard_remaining_duration.as_millis() == Default::default() {
+			if shard_remaining_duration.as_millis() == u128::default() {
```

Comment:
Just curious: What was the reason for this change?

Reply:
I would love to know myself. I don't know... One day clippy (make was fine) didn't compile anymore because `==` didn't know the type or something. 
I'm guessing somewhere somehow a std / sgx feature import is missing, so something wasn't imported full scale. (Which is why so many Cargo.tomls have been changed.. I was trying to find the error source, but I gave in in the end..)

Reply:
Okay, well that's already a sufficient answer for me - clippy is always the boss